tools/nav.txt

```diff
@@ -38,15 +38,15 @@
 +models.html@Models
```

Comment:
This file isn't used any more, so thanks for reminding me it exists, I can delete it now